pkg/kubectl/cmd/exec/exec.go

```diff
@@ -330,3 +381,12 @@ func (p *ExecOptions) Run() error {
 
 	return nil
 }
+
+func (p *ExecOptions) findExecutablePod(obj runtime.Object) (*corev1.Pod, error) {
```

Comment:
I don't see any reason you need this method, for testing you can inject `ExecutablePodFn` into `ExecOptions`.

Reply:
@prksu please address this comment, thanks

Reply:
yes soltysh is right about that. `findExecutablePod` method is not necessary. I just updated my commit.